sql/pgtap--1.3.3--1.3.4.sql

```diff
@@ -1 +1,71 @@
--- TBD
+-- index_is_partial( schema, table, index, description )
+CREATE OR REPLACE FUNCTION index_is_partial ( NAME, NAME, NAME, text )
+RETURNS TEXT AS $$
+DECLARE
+    res boolean;
+BEGIN
+    SELECT x.indpred IS NOT NULL
+      FROM pg_catalog.pg_index x
+      JOIN pg_catalog.pg_class ct    ON ct.oid = x.indrelid
+      JOIN pg_catalog.pg_class ci    ON ci.oid = x.indexrelid
+      JOIN pg_catalog.pg_namespace n ON n.oid = ct.relnamespace
+     WHERE ct.relname = $2
+       AND ci.relname = $3
+       AND n.nspname  = $1
+      INTO res;
+
+      RETURN ok( COALESCE(res, false), $4 );
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
Seems like this can be a pure SQL function:

```suggestion
CREATE OR REPLACE FUNCTION index_is_partial ( NAME, NAME, NAME, text )
RETURNS TEXT AS $$
    SELECT ok(x.indpred IS NOT NULL, $4)
      FROM pg_catalog.pg_index x
      JOIN pg_catalog.pg_class ct    ON ct.oid = x.indrelid
      JOIN pg_catalog.pg_class ci    ON ci.oid = x.indexrelid
      JOIN pg_catalog.pg_namespace n ON n.oid = ct.relnamespace
     WHERE ct.relname = $2
       AND ci.relname = $3
       AND n.nspname  = $1;
$$ LANGUAGE sql;
```